mvvmcDevSprint/Controllers/EuclidesSena/Login/ESLoginViewController.swift

```diff
@@ -21,9 +25,21 @@ class ESLoginViewController: UIViewController {
     var yVariation: CGFloat = 0
     var textFieldIsMoving = false
     
+    private var viewModel: ESLoginViewModelProtocol
```

Comment:
private let

Reply:
Feito

Reply:
Se eu colocar let ele não deixa eu fazer o self no delegate dentro do viewDidLoad()

Cannot assign to property: 'viewModel' is a 'let' constant

Reply:
faz o delegate = self na factory, é melhor configurar por lá

Reply:
Feito